docs/pages/store/reference/store-core.mdx

```diff
@@ -844,3 +844,1328 @@ event Store_DeleteRecord(ResourceId indexed tableId, bytes32[] keyTuple);
 | ---------- | ------------ | ------------------------------------------------------- |
 | `tableId`  | `ResourceId` | The ID of the table where the record is deleted.        |
 | `keyTuple` | `bytes32[]`  | An array representing the composite key for the record. |
+
+## StoreCoreInternal
+
+[Git Source](https://github.com/latticexyz/mud/blob/main/packages/store/src/StoreCore.sol)
+
+_This library contains internal functions used by StoreCore.
+They are not intended to be used directly by consumers of StoreCore._
+
+### State Variables
+
+#### SLOT
+
+```solidity
+bytes32 internal constant SLOT = keccak256("mud.store");
+```
+
+#### DYNAMIC_DATA_SLOT
+
+```solidity
+bytes32 internal constant DYNAMIC_DATA_SLOT = keccak256("mud.store.dynamicData");
+```
+
+#### DYNAMIC_DATA_LENGTH_SLOT
+
+```solidity
+bytes32 internal constant DYNAMIC_DATA_LENGTH_SLOT = keccak256("mud.store.dynamicDataLength");
+```
+
+### Functions
+
+#### \_spliceDynamicData
+
+SET DATA
```

Comment:
This is coming from the code separator comments, we might have to filter this out in the post processing.

Reply:
You mean the "SET DATA" line, right?

Reply:
yep!